UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common.UnitTests/Helpers/NMHelperTest.cs

```diff
@@ -237,6 +240,14 @@ public void WhenValidateParametersForDownloadSingleFileIsCalled_ThenShouldNotThr
             Assert.DoesNotThrow(() => NMHelper.ValidateParametersForDownloadSingleFile(parameters, string.Empty, fakeLogger));
         }
 
+        [Test]
+        public async Task WhenGetFileBytesFromStreamIsCalled_ThenShouldReturnByte()
+        {
+            Stream stream = new MemoryStream(Encoding.UTF8.GetBytes("test stream"));
+            var result = await NMHelper.GetFileBytesFromStream(stream);
```

Comment:
Use explicit type instead of var.
```suggestion
            byte[] result = await NMHelper.GetFileBytesFromStream(stream);
```

Reply:
done